templates/theme/_base.scss

```diff
@@ -0,0 +1,90 @@
+/*
+  WET-BOEW
+  @title: Theme template
+  @desc:  Styles specific to the Theme template for GCWeb
+*/
+
+.page-type-theme {
+	#wb-bc {
+		li:first-child a {
+			border-left: solid #26374a 5px;
+			padding-left: 8px;
+		}
+
+		.breadcrumb {
+			margin-bottom: 15px;
+		}
+	}
+
+	#gridContainer {
+		> nav {
+			#menuTrigger {
+				border-radius: 0;
+				display: block;
+				margin-bottom: 1em;
+				margin-left: -15px;
+				text-align: left;
+				width: calc(100% + 30px);
+
+				.glyphicon {
+					margin-left: 10px;
+				}
+
+				&[aria-expanded=true] .glyphicon {
```

Comment:
Humm, on ne devrait pas utiliser des sélecteur ARIA pour le style car j'ai appris que cela n'est pas une bonne pratique. Il faudrait utiliser une class CSS à la place.

Reply:
Changed to add a class "expanded".